packages/cli-utils/src/concurrency.ts

```diff
@@ -0,0 +1,12 @@
+import { cpus } from 'os'
+
+export function getConcurrencyLimit (option: number | undefined): number {
+  if (typeof option === 'number') {
+    if (option > 0) return option
+
+    // If option is <= 0, it returns the amount of cores plus the number given (since negative: substraction), returning at least 1
+    return Math.max(1, cpus().length + option)
```

Comment:
according to the comment, it should be `- option`. Or the comment should be different?

Reply:
the idea of this is that you can set "-2" and it will use all the cores of the host minus 2 (this convention is used in some other places, as I vaguely remember), but since the option itself is negative, it should be (+ (-2)), if it's minus it would be (- (-2)) = (nCores) + 2, maybe the comment I didn't explain it that well

Reply:
ok, sounds reasonable

Reply:
changed the logic to "return Math.max(1, cpus().length - Math.abs(option))", It will basically do the same, but maybe it's more readable, and the tradeoff of calling an extra Math.abs function shouldn't matter